packages/cardano-graphql/test/e2e/projection.test.ts

```diff
@@ -0,0 +1,20 @@
+import { DgraphClient, createDgraphClient } from '../../src';
+import fs from 'fs';
+import path from 'path';
+
+const schema = fs.readFileSync(path.resolve(__dirname, '..', '..', 'dist', 'schema.graphql'), 'utf-8');
+const address = 'localhost:9080';
+
+describe('Projections', () => {
+  let dgraphClient: DgraphClient;
+
+  beforeEach(() => {
+    dgraphClient = createDgraphClient(address, console);
+  });
+
+  describe('setting the schema', () => {
+    it('completes without throwing an error', async () => {
+      await expect(dgraphClient.setSchema(schema)).resolves;
```

Comment:
This is actually failing with `Error: 2 UNKNOWN: while lexing...`

![image](https://user-images.githubusercontent.com/303881/147203785-52087dfd-0175-4c99-b5d9-4ca50bb139d9.png)